tensorflow/training/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -104,16 +105,30 @@ RUN cd /tmp \
   && make -j64 src.build BUILDDIR=/usr/local \
   && rm -rf /tmp/nccl
 
-# Install EFA along with AWS OPEN_MPI
+# Install EFA along without AWS OPEN_MPI
```

Comment:
With this, will we still require `RDMAV_FORK_SAFE=1` to be set on the dockerfile's env variables?

Reply:
`RDMAV_FORK_SAFE` is required if a program that uses EFA needs to be able to call `fork`. Given our change to MPI, programs that only use MPI doesn't need this flag anymore. But programs that use NCCL or Herring will still need this. If we don't set this flag, programs that use NCCL or Herring and call `fork` will crash.